README.md

```diff
@@ -1,5 +1,9 @@
 ### WRF-ARW Modeling System  ###
 
+This is the WRF feature branch for coupling WRF with CTSM [The Community Terrestrial Systems Model](https://github.com/ESCOMP/CTSM).
```

Comment:
If this is going to merge onto the develop branch, then it is no longer a feature branch. Can you make a change here? Perhaps the change in this file is no longer necessary?

Reply:
You are right. I've just removed these changes in README.md.